src/mudlet.cpp

```diff
@@ -209,7 +209,13 @@ mudlet::mudlet()
     setAttribute(Qt::WA_DeleteOnClose);
     QSizePolicy sizePolicy(QSizePolicy::Expanding, QSizePolicy::Expanding);
     setWindowTitle(version);
-    setWindowIcon(QIcon(QStringLiteral(":/icons/mudlet_main_48px.png")));
+    if (scmIsReleaseVersion) {
```

Comment:
Can I suggest you swap the test around - so that *dev* builds keep with the release icon until a second test produces a functional icon - at present this will give the development one the *ptb* one which is probably not what you intend...

Reply:
Actually I was expecting that dev image momentarily.. do you have any ETA on that?